app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -27,15 +33,14 @@ def needs_proof_of_address?
         !enrollment.current_address_matches_id
       end
 
+      def partner_agency
+        enrollment.issuer
```

Comment:
i'm thinking through this, it definitely makes sense to ensure that the name is easy to read.Looking at what's happening w/ the example you linked to `sp_name` gets passed to the presenter (and the name is retrieved from the session). What i'm thinking through right now is this presenter is used for 2 emails, one of which doesn't need the sp_name - i guess it can be an optional param or we could add a method that gets the friendly name using the issuer.

Reply:
Oh, we do something similar in the "You successfully verified with IPP" email. We pull the human-friendly service provider name off of the enrollment. Here is where it's [used in the email](https://github.com/18F/identity-idp/blob/7adaaef00bba03420928cc81729e40ac3e6e2b3e/app/views/user_mailer/in_person_verified.html.erb#L20) and then here's where it's [defined in the presenter](https://github.com/18F/identity-idp/blob/41dc8580d28548e79b0c3aa6ae6792b1939f4c5c/app/presenters/idv/in_person/verification_results_email_presenter.rb#L26-L28).

Reply:
yup i think that matches what we're trying to do here. i think what's tripping me up when it comes to the enrollment model is that i don't see service_provider as part of the schema and the model says the enrollment belongs to the service_provider. Does the service_provider conversely belong to the enrollment?

Reply:
Yeah Rails magically adds that link in both directions

Reply:
That particular relationship is kind of unusual because [we use the `issuer` as the foreign key](https://github.com/18F/identity-idp/blob/fd3a55f8abad297807fe6f1cd89d7039966d854a/db/schema.rb#L646) rather than having something like `service_provider_id`

Reply:
ahh ok interesting. rails magic definitely throws me off at times.